fabric-keybinding-api-v1/src/main/java/net/legacyfabric/fabric/mixin/keybinding/KeyBindingAccessor.java

```diff
@@ -25,8 +26,8 @@
 
 @Mixin(KeyBinding.class)
 public interface KeyBindingAccessor {
-	@Accessor("categoryOrderMap")
-	static Map<String, Integer> fabric_getCategoryMap() {
-		throw new AssertionError();
+	@Accessor
+	static Map<String, Integer> getCATEGORIES() {
```

Comment:
Why not pass the field to the annotation and have a not so cursed method name?

Reply:
why so many reviews on code style rather than code

Reply:
cause yes